go/libkb/chain_link_v2.go

```diff
@@ -197,6 +197,11 @@ type OuterLinkV2 struct {
 	// - it can be stubbed for non-admins
 	// - it cannot be stubbed for admins
 	IgnoreIfUnsupported SigIgnoreIfUnsupported `codec:"ignore_if_unsupported"`
+	// -- Links exist in the wild that are missing fields below this line too.
+	// If not provided, both of these are nil, and hPrevInfo in the inner link is set to nil.
+	// Note that a link providing HPrevSeqno != nil and HPrevHash == nil is valid for an initial link.
```

Comment:
Why is `HPrevSeqno != nil and HPrevHash == nil` valid? Is it required? I think it would be ideal if there were only one possible representation for an initial link (enforced by server).

Reply:
huh. yes. i agree. an initial link (team root and subteam head) should always be `HPrevSeqno==0` and `HPrevHash==nil`. and if we're going to force a full reload on pre-existing teams before new links can be posted, then these should be the only cases that have HPrevHash of nil moving forward.

Reply:
Yes, if hash is null only seqno=0 is valid. This is checked in VerifyChain, the outerlink stuff just checks that inner/outer match.